src/Toast.js

```diff
@@ -89,7 +89,15 @@ const Toast = React.forwardRef(
       onCloseRef.current();
     }, [autohide, show]);
 
-    autohideTimeout.set(autohideFunc, delayRef.current);
+    useEffect(() => {
+      if (autohide && show) {
+        autohideTimeout.set(autohideFunc, delayRef.current);
+      }
+
+      return () => {
+        autohideTimeout.clear();
+      };
```

Comment:
```suggestion
      autohideTimeout.set(autohideFunc, delayRef.current);
```

this might be sufficient? `autohideFunc` won't do anything if `autohide && show` isn't enabled, and any re-run of the effect will reset the timeout anyway.

Reply:
This is much cleaner.  Thanks!